File: net/NetTupleEngine/VarToken.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace it.mintlab.desktopnet.tupleengine
{
    class VarToken : IToken
    {
        private string var;

        public VarToken(string var)
        {
            this.var = var;
        }

        public override string ToString()
        {
            return var;
        }


    }
}

File: trunk/java/JavaTupleEngine/src/it/mintlab/desktopjava/tupleengine/VarToken.java
package it.mintlab.desktopjava.tupleengine;

	public class VarToken implements IToken
	{
		private String var;

		public VarToken(String var)
		{
			this.var = var;
		}

		@Override
		public String toString()
		{
			return var;
		}


	}